fuzzers/weizz_qemu/fuzzer.py

```diff
@@ -40,6 +40,13 @@ def fuzz(input_corpus, output_corpus, target_binary):
     os.environ['WEIZZ_NO_AFFINITY'] = '1'
     os.environ['WEIZZ_I_DONT_CARE_ABOUT_MISSING_CRASHES'] = '1'
     os.environ['WEIZZ_CTX_SENSITIVE'] = '1'
+
+    # Weizz needs at least one non-empty seed to start.
+    if len(os.listdir(input_corpus)) == 0:
+        with open(os.path.join(input_corpus, 'default_seed'),
```

Comment:
I'd do the os.path.join on another line to avoid the ugly line break.

Reply:
Done! Also, using a helper now for both afl and weizz.